### ISSUE TITLE
overlaps between training set and valdiation set

### ISSUE THREAD

**Issue by buptxiaofeng** [abdoelsayed2016/TNCR_Dataset#3]
Thanks for sharing this impressive work. I compared the training set and validation set, and there are 28 overlap images. What's the purpose of these overlap images?

**Comment by abdoelsayed2016** [comment]
maybe mistake while we splitting the dataset. you should remove them from the training or validation

**Comment by buptxiaofeng** [comment]
thanks for the reply. I close this issue.
